xbmc/music/MusicEmbeddedCoverLoaderFFmpeg.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "cores/FFmpeg.h"
```

Comment:
For what do you need this include? It should not be needed.

Reply:
Please remove the include, insert forward declaration `struct AVFormatContext;` instead.

Reply:
Add `#include "cores/FFmpeg.h"` to the cpp file.